cloud/aws-cloudwatch-metric-stream/policies.tf

```diff
@@ -0,0 +1,77 @@
+data "aws_iam_policy_document" "sfx_firehose_policy" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["firehose.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_policy" "sfx_firehose_policy" {
```

Comment:
It does not seem essential in a final module as here but using https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document can allow us to enjoy the native terraform error checking and formatting capabilities.

Reply:
Good point, done!